win32ss/user/user32/misc/exticon.c

```diff
@@ -323,6 +323,44 @@ static UINT ICO_ExtractIconExW(
 	}
 	CloseHandle(fmapping);
 
+#ifdef __REACTOS__
+    /* Check if the resource is an animated icon/cursor */
+    if (!memcmp(peimage, "RIFF", 4))
+    {
+        INT     anihOffset;
+        ULONG   uSize = 0;
+
+        /* Get size of the animation data */
+        uSize = (WORD)peimage[4] +
+                256 * (WORD)peimage[5];
```

Comment:
It would be more common to write `(WORD)peimage[5] << 8`

Reply:
Done. Thanks.